### DISCUSSION TITLE
minimize button on fullscreen menubar

### DISCUSSION THREAD

**Discussion by test2a** [rustdesk/rustdesk#4677] [Feature Request]
I do not like using the fullscreen menubar. there is a problem.

when we use windows RDP, it has a close and minimize button so one can use local machine and go to fullscreen and back.

rustdesk on the other hand makes it 3 step process.

 you are in fullscreen mode.

1. you press on the menu icon to bring the menu bar.
2. you press on restore windows button
3. you press minimize button

3 steps which can be avoided if we just had a minimize button on the fullscreen menubar itself because when we are in windowed mode, we can see minimize button on the top right corner alongwith close but not when fullscreen is enabled.

**Comment by lukaszjablonski** [comment]
A quick mockup:
![RustDesk-MinimiseBtn-Mockup](https://github.com/rustdesk/rustdesk/assets/13468012/ebdda07b-9c78-4562-a8ff-6fcc17b2d510)
